heroclass_tina.rb

```diff
@@ -0,0 +1,109 @@
+class Superhero
+  
+   $heroes = [{name: "Batman", identity: "Bruce Wayne", weakness: "Woman"},
+  {name: "Spiderman", identity: "Peter Parker", weakness: "Mary Jane"},
+  {name: "Superman", identity: "Clark Kent", weakness: "Louis Lane"}]
+    input = ''
+  @@herohash = {}
+  def Superhero.get_users
+    @@herohash
+  end
+
+  def deleted_hero
+    puts "What is the name of the superhero that has passed?"
+    name = gets.chomp
+    $heroes.reject! {|hero| hero[:name].downcase == name.downcase}
+    puts "I am sorry #{name} has passed ...you do have the body right? Better get that shovel and start digging!"
+    puts
+  end
+
+  def list_hero      
+        puts
+        puts" Here is a list of all the superheroes."
+      $heroes.each do |hero|
+        puts "      =======================
+        Superhero: #{hero[:name]}"
+    end
+        puts"      ======================="
+        puts
+  end
+
+  def list_hero_complete
+        puts
+        puts" Here is a list of all the superheroes, secret identities and weeknesses."
+      $heroes.each do |hero|
+        puts "        ===============================
+          Superhero:         #{hero[:name]}
+          Secret Identity:   #{hero[:identity]}
+          Weakness:          #{hero[:weakness]}"
+        end
+  puts"        ==============================="
+  puts
+  end
+
+  def list_heros_secret_identity
+      puts "Please enter your login information"
+      login = gets.chomp.to_s
+    if login == "HeroesRock"
+      puts list_hero_complete
+        else 
+      puts "That is the incorrect password"
+      list_heros_secret_identity
+      end
+  end
+end
+shero = Superhero.new
```

Comment:
I would suggest using super_hero as the variable name, this is part of the best practices of variable naming.